cadl-tests/src/test/java/com/cadl/testserver/servicedriven2/DPGClientTest.java

```diff
@@ -12,7 +12,7 @@
 import org.junit.jupiter.api.Test;
 
 class DPGClientTest {
-    DPGClient client = new DPGClientBuilder().buildClient();
+    ResiliencyServiceDriven2Client client = new ResiliencyServiceDriven2ClientBuilder().buildClient();
```

Comment:
@haolingdong-msft 

The updated cadl-ranch throws 404 ResourceNotFoundException in runtime, please take a look.

I might force merge this PR, as none of it changes codegen.

Reply:
will check